observations/r/__init__.py

```diff
@@ -0,0 +1,2308 @@
+"""[Observations.r](https://github.com/edwardlib/observations) provides
```

Comment:
Actually, I just noticed this file is `r/__init__.py` and not `__init__.py`. You can remove this docstring. Alternatively, can alter it to describe that this submodule consists of R data sets.

Reply:
fixed